geonode/qgis_server/views.py

```diff
@@ -740,3 +740,35 @@ def set_thumbnail(request, layername):
     }
     return HttpResponse(
         json.dumps(retval), content_type="application/json")
+
+
+def download_qlr(request, layername):
+    """Download QLR file for a layer
```

Comment:
We usually add a full stop at the end of the line. I'm not sure if we do that in geonode too, but we should (PEP257) ;-)